core/src/main/java/org/apache/accumulo/core/spi/balancer/SimpleLoadBalancer.java

```diff
@@ -66,15 +66,9 @@ public class SimpleLoadBalancer implements TabletBalancer {
   protected BalancerEnvironment environment;
 
   Iterator<TabletServerId> assignments;
-  // if tableToBalance is set, then only balance the given table
-  TableId tableToBalance = null;
 
   public SimpleLoadBalancer() {}
 
-  public SimpleLoadBalancer(TableId table) {
-    tableToBalance = table;
-  }
-
```

Comment:
Just adding context here: this constructor wasn't added to comply with the TabletBalancer SPI, but to make this implementation supportable by the delegating per-table balancer named TableLoadBalancer.

The whole reason TableLoadBalancer exists is to delegate to other balancers on a per-table basis. If that is no longer needed, because per-table delegation is now built-in at a higher level, then the whole TableLoadBalancer can be deleted... it is not needed anymore.